pkg/git/libgit2/managed_test.go

```diff
@@ -35,12 +36,16 @@ import (
 	"github.com/go-logr/logr"
 	. "github.com/onsi/gomega"
 
+	git2go "github.com/libgit2/git2go/v33"
 	cryptossh "golang.org/x/crypto/ssh"
-	corev1 "k8s.io/api/core/v1"
 )
 
 const testRepositoryPath = "../testdata/git/repo"
 
+func TestMain(m *testing.M) {
+	managed.InitManagedTransport(logr.Discard())
+}
```

Comment:
`TestMain()` not calling `m.Run()` results in none of the tests in this whole package to run.